src/main/webapp/student/enrollment/schoolClass/schoolClassesSelection.jsp

```diff
@@ -0,0 +1,206 @@
+<%@ taglib uri="http://struts.apache.org/tags-html" prefix="html" %>
+<%@ taglib uri="http://struts.apache.org/tags-bean" prefix="bean" %>
+<%@ taglib uri="http://struts.apache.org/tags-logic" prefix="logic"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<link href="${pageContext.request.contextPath}/CSS/dotist_timetables.css" rel="stylesheet" type="text/css" />
+
+<link href='${pageContext.request.contextPath}/themes/fenixedu-learning-theme/static/css/fullcalendar.css' rel='stylesheet' />
```

Comment:
The development was based on the fenixedu-learning-theme example provided by Pedro.

Since fenixedu-academic module doesn't have fullcalendar installed, should I put it in webapp (or another) folder?